crates/hyperqueue/src/worker/hwdetect.rs

```diff
@@ -58,12 +62,26 @@ pub fn detect_generic_resources() -> anyhow::Result<Vec<GenericResourceDescripto
     }
 
     if let Ok(mem) = read_linux_memory() {
-        log::debug!("Mem detected: {}", mem);
+        log::info!("Detected {mem}B of memory ({})", human_size(mem));
```

Comment:
``human_size`` already adds units

Reply:
The `B` is added to the raw number, not to the formatted `human_size` output.